### ISSUE TITLE
[Feature Request] Browser extensions

### ISSUE THREAD

**Issue by XanderStrike** [m1k1o/neko#114]
When attempting to install extensions the following occurs:

![this plugin is essential to my workflow](https://user-images.githubusercontent.com/1565303/145115052-a2a80370-07da-4429-a4a2-6752ba8e23f7.png)

~~Looks like this is probably a result of [the following lines](https://github.com/m1k1o/neko/blob/4a1d3944c5274fb3613e9ee18d67510e12c6f464/.docker/firefox/neko.js#L18-L19).~~ There may be a good reason for disabling it but the commit message doesn't have any hints.

I may remove those lines and test it out to see what happens. I expect plugins will be wiped on container restart which is fine for my purposes, but there might be another feature request for storing the firefox user profile in a volume to make them persist.

**Comment by m1k1o** [comment]
No, it's here. For security, if the browser would be compromised, installing any extension could potentially allow anyone to access filesystem or steal all cookies, inject malware to any page... 

https://github.com/m1k1o/neko/blob/4a1d3944c5274fb3613e9ee18d67510e12c6f464/.docker/firefox/policies.json#L56-L58

**Comment by XanderStrike** [comment]
True, but that's not much of a risk for my use case since my instance of Neko is only accessible to me through a VPN. I expect lots of folks are running it password protected behind a reverse proxy with TLS too. Perhaps it shouldn't be on by default, but it'd be nice to have the option. 

As an alternative maybe additional blocks [like these](https://github.com/m1k1o/neko/blob/4a1d3944c5274fb3613e9ee18d67510e12c6f464/.docker/firefox/policies.json#L59-L66) could be passed in as configuration to allow the operator to hardcode which extensions should be installed, that would satisfy my needs.

**Comment by m1k1o** [comment]
That would be a nice usecase, i agree. But that would mean, we would need some shell that runs before browser, creates that environment (modifies that file according to e.g. env variables) and starts browser. It can be done, but another way is just to mount that file to the container and modify according to your needs. That's how it is often done.

See section `Want to customize and install own add-ons, set custom bookmarks?`
https://neko.m1k1o.net/#/getting-started/?id=want-to-customize-and-install-own-add-ons-set-custom-bookmarks

**Comment by m1k1o** [comment]
Added to docs.
